tests/10apidoc/30room-create.pl

```diff
@@ -366,6 +366,9 @@ =head2 room_alias_fixture
 An optional prefix string can be provided, which will be prepended onto the
 alias name.
 
+An optional remote boolean can be supplied, which will generate an alias for
+the remote homeserver instead.
```

Comment:
I don't think we really talk about the "remote" homeserver anywhere else.

```suggestion
the second homeserver instead.
```